framework/packages/abstract-client/src/account.rs

```diff
@@ -549,7 +549,7 @@ impl<Chain: CwEnv> Account<Chain> {
 
     /// Set IBC status on an Account.
     pub fn set_ibc_status(&self, enabled: bool) -> AbstractClientResult<()> {
-        self.abstr_account.manager.update_settings(Some(enabled))?;
+        self.abstr_account.manager.set_ibc_status(enabled)?;
 
         Ok(())
     }
```

Comment:
Do we need `set_ibc_status` method? I like that it's clear that it can be enabled or disabled any time on account. Should we have `toggle_adapter` instead?

Reply:
You feel like `set_ibc_status` doesn't convey that it can be enabled and disabled ?

Reply:
Yes that's a good thought !

Reply:
As a helper method yes could be nice!